scripts/release.ts

```diff
@@ -81,16 +82,13 @@ step("\nUpdating package version...");
 updateVersion(pkgPath, targetVersion);
 
 step("\nGenerating changelog...");
-const changelogArgs = [
-  "conventional-changelog",
-  "-p",
-  "angular",
-  "-i",
-  "CHANGELOG.md",
-  "-s",
-  "--commit-path",
-  ".",
-];
+const latestTag = await getLatestTag(pkgName);
+if (!latestTag) process.exit();
+const sha = await run("git", ["rev-list", "-n", "1", latestTag], {
+  stdio: "pipe",
+}).then((res) => res.stdout.trim());
```

Comment:
if you're awaiting then you don't need to use a `.then`

Reply:
Need to do this since the await returns ExecReturnValue<string>